datasets/SWiFT/temperature_profile_reconstruction.ipynb

```diff
@@ -0,0 +1,1128 @@
+{
```

Comment:
<p>Do you have a reference for this formula? If so, might be interesting to add it here</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/a2e-mmc/assessment/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>